packages/viz/src/utils/index.ts

```diff
@@ -201,3 +204,40 @@ export const getHvArqueroCombinedFilters = (
     return filterFunction(row);
   });
 };
+
+/**
+ * Normalizes a column name by replacing all characters that are not letters or numbers by "___"
+ */
+export const normalizeColumnName = (string: string) => {
+  return string.replace(/[^a-zA-Z0-9]/g, "___");
```

Comment:
I think I may have exaggerated with the number of `_` 😅
I didn't want to just add one `_` to avoid having duplicated columns. It seems pretty common to have column names like `sales_target`, so in case a user has a table with the columns `sales_target` and `sales target`, `sales target` gets normalised to `sales___target` instead of `sales_target`. I'm probably overthinking this but it didn't seem to hurt 😅
But I'll switch to 2 `_`, I got carried away with 3 😬

Reply:
hehe ok 😄